packages/maskbook/package.json

```diff
@@ -171,5 +171,8 @@
   },
   "peerDependencies": {
     "jss": "*"
+  },
+  "volta": {
```

Comment:
What is this?

Reply:
@Jack-Works this is my node version manager, it shouldn't be there. lol